environment.yml

```diff
@@ -1,14 +1,7 @@
 name: mirflowz
 channels:
-  - bioconda
   - conda-forge
-  - defaults
+  - bioconda
```

Comment:
Put `bioconda` first - the order matters. And we want `Snakemake` from `bioconda`.